db/queries/seller.sql

```diff
@@ -0,0 +1,404 @@
+-- name: SellerGetInfo :one
+
+SELECT
+    "seller_name",
+    "image_id",
+    "description",
+    "enabled"
+FROM "shop"
+WHERE "seller_name" = $1;
+
+-- name: SellerUpdateInfo :one
+
+UPDATE "shop"
+SET
+    "image_id" = COALESCE($2, "image_id"),
+    "name" = COALESCE($3, "name"),
+    "description" = COALESCE($4, "description"),
+    "enabled" = COALESCE($5, "enabled")
+WHERE "seller_name" = $1
+RETURNING
+    "seller_name",
+    "image_id",
+    "name",
+    "enabled";
+
```

Comment:
Since we can get `username` by middleware, consider use that instead of re-query again.